tests/rptest/services/openmessaging_benchmark_worker.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import re
+import threading
+import os
+
+from ducktape.services.service import Service
+from ducktape.utils.util import wait_until
+
+
+# worker need to be started near redpanda process #ToDo: fix comment
+class OpenMessaging_Benchmark_Worker():
+    PERSISTENT_ROOT = "/var/lib/openmessaging"
+    STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "worker.log")
+    PORT = 9090
+    STATS_PORT = 9091
+
+    def __init__(self, redpanda):
+        self.redpanda = redpanda
+        for node in self.redpanda.nodes:
+            self.start_node(node)
+
+    def start_node(self, node):
+        self.redpanda.logger.error(
```

Comment:
once this inherits from Service abstraction it should have its own logger context